src/java.desktop/share/classes/javax/swing/plaf/basic/BasicComboBoxUI.java

```diff
@@ -1118,35 +1118,33 @@ else if (comboBox.getModel().getSize() > 0) {
     // This is currently hacky...
     @Override
     public int getAccessibleChildrenCount(JComponent c) {
-        if ( comboBox.isEditable() ) {
-            return 2;
-        }
-        else {
-            return 1;
-        }
+        return 2;
     }
 
     // This is currently hacky...
     @Override
     public Accessible getAccessibleChild(JComponent c, int i) {
         // 0 = the popup
-        // 1 = the editor
+        // 1 = the editor for editable combobox and the arrow button for non-editable combobox
         switch ( i ) {
-        case 0:
-            if ( popup instanceof Accessible ) {
-                AccessibleContext ac = ((Accessible) popup).getAccessibleContext();
-                ac.setAccessibleParent(comboBox);
-                return(Accessible) popup;
-            }
-            break;
```

Comment:
Why do you change indentation? To minimise the number of changed lines, please keep the existing indentation.

Reply:
It's a matter of style.

The [old conventions](https://www.oracle.com/docs/tech/java/codeconventions.pdf)¹ suggest that `case` statements shouldn't be indented. The code followed this style.

[A latest draft](https://cr.openjdk.org/~alundblad/styleguide/index-v6.html) which is often referred to explicitly states that `case` statements should be indented with four spaces and the statements inside should be be indented with additional four spaces.

The updated code follows the new style. Since you're updating the code in the `switch` statement, it's fine to change the indentation… provided we agree to *follow the new style*.

¹ The [HTML version for `switch` statement](https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#468) is completely *unreadable now*.

Reply:
> It's a matter of style.
> 
> The [old conventions](https://www.oracle.com/docs/tech/java/codeconventions.pdf)¹ suggest that `case` statements shouldn't be indented. The code followed this style.
> 
> [A latest draft](https://cr.openjdk.org/~alundblad/styleguide/index-v6.html) which is often referred to explicitly states that `case` statements should be indented with four spaces and the statements inside should be be indented with additional four spaces.
> 
> The updated code follows the new style. Since you're updating the code in the `switch` statement, it's fine to change the indentation… provided we agree to _follow the new style_.
> 
> ¹ The [HTML version for `switch` statement](https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#468) is completely _unreadable now_.

The indented code is more readable and as you pointed it is inline with new style too. So, I am ok with the current indentation if we are on same page.